### ISSUE TITLE
Publish a contribution policy

### ISSUE THREAD

**Issue by nkeim** [soft-matter/trackpy#37]
It would be helpful, sooner rather than later, to have guidelines for version numbering, PR etiquette, etc. on the Github site. Perhaps we can just open a project wiki and slap it up on the front page.

The actual content could be along the lines of Tom's email on 1/28. Most salient point: don't merge your own PR.

**Comment by nkeim** [comment]
Finally: https://github.com/soft-matter/trackpy/wiki/Information-for-contributors . Please have a look. Too wordy? Too lax? Too strict?

Also, we now have a "core contributors" section on the [introduction page](http://soft-matter.github.io/trackpy/introduction.html). Good enough for now.

**Comment by tacaswell** [comment]
:+1:

**Comment by danielballan** [comment]
Yep, looks great. Thanks for taking the time to put this together.

**Comment by nkeim** [comment]
Great! Please edit as inspiration strikes.
